src/components/GeoPushpinChart.tsx

```diff
@@ -11,6 +11,7 @@ import { convertBucketsToAFM, convertBucketsToMdObject } from "../helpers/conver
 import { getGeoChartDimensions } from "../helpers/dimensions";
 import { getResultSpec } from "../helpers/resultSpec";
 import { IGeoPushpinChartProps } from "./../interfaces/GeoChart";
+import { disableBucketItemComputeRatio } from "../helpers/optionalStacking/common";
```

Comment:
should we move this function to a general file other than `optionalStacking`?

Reply:
this function was introduced by other member and used at other charts.

Reply:
I agree with Quang, we can move `disableBucketItemComputeRatio ` to./src/helpers/utils.ts`
If refactoring makes code look better, we should do it.

Reply:
fixed